static/scripts/management.js

```diff
@@ -0,0 +1,55 @@
+const MAX_FILE_SIZE_MB = 4;
+
+const toBase64 = (file) => new Promise((resolve, reject) => {
+	const reader = new FileReader();
+	reader.readAsDataURL(file);
+	reader.onload = () => resolve(reader.result);
+	reader.onerror = (error) => reject(error);
+});
+
+const loadPolicyFile = () => {
+	const file = document.querySelector('#policy-input').files[0];
+	toBase64(file).then((base64file) => {
+		const reader = new FileReader();
+		reader.addEventListener('load', (evt) => {
+			if (!file.type.match('application/pdf')) {
+				$.showNotification('nur PDF Dateien werden unterstützt', 'danger', true);
```

Comment:
i18n would be nice here too as in all places with hardcoded messages.

Reply:
we do not use i18n in SHD platofrm :)